Admin/ArticleAdmin.php

```diff
@@ -258,7 +264,6 @@ protected function configureShowFields(ShowMapper $showMapper)
             ->add('title', 'text')
             ->add('subtitle', 'text')
             ->add('abstract', 'textarea')
-            ->add('type', 'text')
```

Comment:
totally true, was a field that we forget to remove, you want I extract it to another PR ?